style.css

```diff
@@ -0,0 +1,53 @@
+body{
+    margin: 0;
+    padding: 0;
+    font-family: Quicksand;
+}
+nav{
```

Comment:
Don't use tagnames for styling. It's generally a bad practice. Always use ```classes``` to add styling to elements. This will make it a lot safer and easier to manage for you in larger projects.